### DISCUSSION TITLE
How do you solve the annoying problem of repeating definitions? How to implement variable reassignment?

### DISCUSSION THREAD

**Discussion by deahhh** [fonsp/Pluto.jl#2032] [Questions about using Pluto]

**Comment by eahenle** [comment]
Use `local` when possible.

```julia
begin
    local fig = Figure()
    local ax = Axis(fig[1, 1])
    ...
    fig
end
```

If you need a lot of variables to be `local`, use `let`:

```julia
let
  fig = Figure()
  ax = Axis(fig[1, 1])
  ...
  fig
end
```

Done like this, you can use `fig` and `ax` in as many cells as you like (but they can only be referenced within the cell where they are defined).
